css/styles.css

```diff
@@ -78,6 +86,426 @@
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+}
+
+.story-btn > span > a {
+  text-decoration: none;
+  background-color: #ff6b00;
+  color: #fff;
+  padding: 1rem;
+}
+
+.projects {
+  margin-top: 1rem;
+  display: grid;
+  grid-template-columns: 1.5rem 1fr 1.5rem;
+  grid-template-rows: 100%;
+}
+
+.project-details {
+  grid-column: 2;
+  background-image: url("../pics/img2.svg");
+  background-repeat: no-repeat;
+  padding: 0.8rem;
+}
+
+.project-title {
+  margin-top: 4.5rem;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 2rem;
+  line-height: 2.75rem;
+  color: #fff;
+}
+
+.support-text {
+  margin-top: 1rem;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: #fff;
+}
+
+.lang-list > li > a {
+  margin-top: 2.75rem;
+  text-decoration: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.5rem;
+  line-height: 2.25rem;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.lang-list {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.pro-language > ul {
+  list-style-type: none;
+  display: flex;
+}
+
+.pro-language > ul > li {
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+
+.pro-btn {
+  grid-column: 2;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #ff6b00;
+}
+
+.languages > ul > li :hover {
+  background-color: #091e42;
+  color: #fff;
+}
+
+.pro-btn > span {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+.pro-btn > span > a {
+  text-decoration: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.about {
+  background-color: #1c1a19;
+  margin-top: 7.5rem;
+  display: grid;
+  grid-template-columns: 1.5rem 1fr 1.5rem;
+  grid-template-rows: 100%;
+  background-image: url("../pics/illustration-about-me-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position-x: right;
+  background-position-y: bottom;
+}
+
+.about-text {
+  grid-column: 2;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 4.5rem;
+  line-height: 5.5rem;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin-top: 7.5rem;
+}
+
+.about-description {
+  grid-column: 2;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 2rem;
+  color: #f4f5f7;
+  margin-top: 1.5rem;
+}
+
+.about-footer {
+  grid-column: 2;
+}
+
+.about-btn {
+  background-color: #ff6b00;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 50%;
+  text-align: center;
+  padding: 1rem;
+}
```

Comment:
- Great job 👏 adding the proper styling to this button but once again please make sure you also follow the button styling for the: :hover and :active (pressed) ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=pSU459atAyqpbx9o-0)) states.

    Here's how it looks like in the design:
      ![firefox_IJ0QIflmz9](https://user-images.githubusercontent.com/104968236/227728511-1ac7dbf8-b803-4ed1-a476-d25fe259edce.png)

Reply:
done :heavy_check_mark: